src/commands/install.yml

```diff
@@ -11,8 +11,8 @@ parameters:
     type: string
     default: ""
     description: >
```

Comment:
I am not sure we should change from latest by default to LTS, or if we do, im not sure if it would qualify as a patch/fix. tough call here.

What is the motivation to switch to LTS by default?

Reply:
This PR is not making the switch, but is just fixing that comment. We discussed this a while ago when we made the change and agreed that the default should be LTS as latest may be unstable. 

The other change where lts is checked against the version parameter is to allows it to be still a valid input if declared explicitly.

Reply:
LTS is the recommended version to be used by Node.js themself. So there's that. I wouldn't call it a patch update for sure though. If we change from the tip of "current" to the tip of "lts" as the default version to install, that would be a major update.

Reply:
> LTS is the recommended version to be used by Node.js themself. So there's that. I wouldn't call it a patch update for sure though. If we change from the tip of "current" to the tip of "lts" as the default version to install, that would be a major update.

The change from latest to LTS was in the 5.0 major release. This is just for the comment :) I used `current LTS` instead of `latest LTS` to try to avoid confusion. What do you think would reosolve your confusion?